iFixit/react-commerce
# Integrate Piwik Pro

## Overview

We want to try out Piwik Pro, so lets load it onto every page. Piwik Pro by default saves stuff into `window._paq`, but our code for matomo already handles this being instantiated so it should work fine.

## QA

Make sure that all pages have Piwik Pro and are sending events fine. Make sure that your `NEXT_PUBLIC_PIWIK_ENV` is set to your desired environment (dev for development, prod for production, anything else for none);

Closes: #1990
